src/test/smart-components/group/principal/__snapshots__/principals.test.js.snap

```diff
@@ -194,18 +194,7 @@ Array [
 ]
 `;
 
-exports[`<GroupPrincipals /> should render correctly with empty members 1`] = `
-<EmptyWithFilter
-  description={
-    Array [
-      "Add a user to configure user access.",
-      "",
-    ]
-  }
-  icon={[Function]}
-  title="There are no members in this group"
-/>
-`;
+exports[`<GroupPrincipals /> should render correctly with empty members 1`] = `null`;
```

Comment:
Is this change correct? I can't see changes to `GroupPrincipals` and this snapshot has changed.

Reply:
No, it isn't, good catch, thank you. There was an outdated EmptyState component name in the expect statement.